.github/check_agda.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
```

Comment:
`set -Eeu -o pipefail` is the one I typically use as sort of standard bash prelude. `-E` ensures that subshells also respect the `-e` flag; `-u` throws on undefined variables; and `-o pipefail` ensures that if any elements in a piped expression fail, the result is failure.

This script isn't doing anything complex but it's nice to be consistent.

Reply:
Thanks!